src/rahul_Hiremath/assignment_15/Ass_15.java

```diff
@@ -0,0 +1,155 @@
+package rahul_Hiremath.assignment_15;
+
+import java.util.Scanner;
+
+public class Ass_15 {
+
+	double ICICIBal = 1000;
+	double HDFCBal = 1000;
+	double BOBBal = 1000;
+	double SBIBal = 1000;
+
+	int ICICCreditCount;
+	int HDFCCreditCount;
+	int BOBCreditCount;
+	int SBICreditCount;
+
+	int ICICDebitCount;
+	int HDFCDebitCount;
+	int BOBDebitCount;
+	int SBIDebitCount;
+
+	static int totalCredit;
+	static int totalDebit;
+
+	void debitOrCredit(String bankName, String operation, double amt) {
+		String ignoreCaseBankName = bankName.toUpperCase();
+		switch (operation) {
+		case "credit":
+			if (ignoreCaseBankName.equals("ICICI")) {
+				ICICIBal = ICICIBal + amt;
+				ICICCreditCount++;
+				totalCredit++;
+				break;
+			} else if (ignoreCaseBankName.equals("HDFC")) {
+				HDFCBal = HDFCBal + amt;
+				HDFCCreditCount++;
+				totalCredit++;
+				break;
+			} else if (ignoreCaseBankName.equals("BOB")) {
+				BOBBal = BOBBal + amt;
+				BOBCreditCount++;
+				totalCredit++;
+				break;
+			} else if (ignoreCaseBankName.equals("SBI")) {
+				SBIBal = SBIBal + amt;
+				SBICreditCount++;
+				totalCredit++;
+				break;
+			} else {
+				System.out.println("Bank name do not match");
+				break;
+			}
+
+		case "debit":
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.